shared/people/task/container.js

```diff
@@ -149,15 +150,16 @@ const teamShowcaseConnector = connect(
 )
 
 export default compose(
-  branch(props => props.todoType === 'bio', bioConnector),
-  branch(props => props.todoType === 'proof', proofConnector),
-  branch(props => props.todoType === 'device', deviceConnector),
-  branch(props => props.todoType === 'follow', followConnector),
-  branch(props => props.todoType === 'chat', chatConnector),
-  branch(props => props.todoType === 'paperkey', paperKeyConnector),
-  branch(props => props.todoType === 'team', teamConnector),
-  branch(props => props.todoType === 'folder', folderConnector),
-  branch(props => props.todoType === 'gitrepo', gitRepoConnector),
-  branch(props => props.todoType === 'teamshowcase', teamShowcaseConnector),
+  // TODO maybe have an object
+  branch(props => props.todoType === todoTypes.bio, bioConnector),
```

Comment:
we should ditch the branches really to solve this problem. can do this later but this is the direction we should go

Reply:
then you can also do the switch thing where you ensure you get every case

Reply:
yeah, that would be cool. I'll include that with any other QA stuff that comes in